cornell-dti/office-hours
# "Question Up" Notification Displays Once

### Inside This PR 


* The "Your question is up!" notification will only display once when the user becomes first in the queue. Upon refreshing, the notification will not display a second time.

### Test Plan 
* Check that the notification displays when becoming first in the queue, and does not display upon refresh
* Check that the notification displays again when becoming first under a different question
* Check that the notification only displays once when multiple Queue Me In tabs are open

### Notes 

* window.localStorage was used over window.sessionStorage to prevent multiple notifications from displaying in case the user has Queue Me In open in several tabs

### Breaking Changes 

- [ ] Database schema change (anything that changes Postgres)
- [ ] I updated existing types in `/src/components/types/`
- [ ] Other change that could cause problems (Detailed in notes)

### Checklist




- [ ] My changes requires a change to the documentation.
